packages/components/components/elvis-pagination/src/react/elvia-pagination.tsx

```diff
@@ -78,6 +80,23 @@ export const Pagination: FC<PaginationProps> = function ({
     }
   }, [value]);
 
+  // Toggle language for aria-label
+  const lang = useLang();
+  useEffect(() => {
+    let label = labelOptionsState.label;
+
+    if (lang === 'en') {
+      label = 'items';
+    }
+
+    const langLabel =
+      lang === 'no'
+        ? `viser ${selectedDropdownValue} ${label} per side`
+        : `showing ${selectedDropdownValue} ${label} per page`;
+
+    setAriaLabel(langLabel);
+  }, [lang, selectedDropdownValue, labelOptionsState.label]);
+
```

Comment:
Won't this override the users `labelOptions` prop if the page in in english?

If a user sends in 
```ts
labelOptions = {
 displaying: "viser",
 label: "målepunkter",
 of: "av",
}
```
Will it remain as "målepunkter" or will it be replaced with "items"?

Reply:
That is actually true, it will override every label in english as items. I'll look into this to make it more dynamic.

Reply:
Nice! Respecting the users props in important 😄